operators/bosh-operator/DirectorService.js

```diff
@@ -849,7 +848,7 @@ class DirectorService extends BaseDirectorService {
       })
       .then(resource => {
         let response = _.get(resource, 'status.response', undefined);
```

Comment:
No need to set default as `undefined` as resolved value itself will be `undefined` in that case